website/content/en/highlights/2021-11-04-remap-dropped-output.md

```diff
@@ -0,0 +1,82 @@
+---
+date: "2021-11-04"
+title: "New `dropped` output on the `remap` transform"
+description: "Sending failed events down a separate pipeline"
+authors: ["lukesteensen"]
+pr_numbers: [9169]
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: "new feature"
+  domains: ["remap"]
+---
+
+Vector 0.18 introduces a new `dropped` output to the `remap` transform. This
+can be used to route events that fail processing down a separate pipeline.
```

Comment:
If we expect to only support logs, we should make that clearer in this highlight.

Also, thanks for proactively writing a highlight for this! I usually end up writing these during the release process so this is welcome.

Reply:
This has been updated to allow metrics too.